### ISSUE TITLE
MathJax and markdown

### ISSUE THREAD

**Issue by rajgoel** [hakimel/reveal.js#702]
Hello,

it appears that the MathJax plugin does not always work as expected when using markdown. I experienced that underscores and some brackets are not always properly processed, e.g

```
<section>
$\bar{X}= \frac{1}{T}\sum^T_{t=1} X_t, \bar{Y}=\frac{1}{T}\sum^T_{t=1} Y_t$
</section>

<section data-markdown  data-separator="^\n===\n$" data-vertical="^\n---\n$">
<script type="text/template">

$\bar{X}= \frac{1}{T}\sum^T_{t=1} X_t, \bar{Y}=\frac{1}{T}\sum^T_{t=1} Y_t$

===

$\bar{X}= \frac{1}{T}\sum^T&#95;{t=1} X&#95;t, \bar{Y}=\frac{1}{T}\sum^T&#95;{t=1} Y&#95;t$

</script>
</section>
```

All three slides should produce the same result, but in the second slide the formula is not processed - probably because the underscore is not provided to MathJax. I experienced similar issue with brackets. By including the HTML-code for the underscore the formula is produced correctly.

It would be great if the MathJax plugin would work without the HTML-code workaround when using markdown.

Thanks for developing such a great presentation software :-)

**Comment by pedritomelenas** [comment]
Similar problems arise while using arrays or matrices. For instance, these two slides should look the same:

				<section>
					<h3>Mathjax support without markdow </h3>

					$$
					\begin{pmatrix}
					1 & 2 \\
					3 & 4
					\end{pmatrix}
					$$
				</section>

				<section data-markdown>
					<script type="text/template">
						### Mathjax support with markdown

						$$
						\begin{pmatrix}
						1 & 2 \\
						3 & 4
						\end{pmatrix}
						$$

					</script>
				</section>

The second just displays the four numbers in a row.

**Comment by pedritomelenas** [comment]
It seems that the problem with arrays is solved if we add three backslashes instead of two in markdown for new row.

				<section>
					<h3>Mathjax support without markdow </h3>

					$$
					\begin{pmatrix}
					1 & 2 \\
					3 & 4
					\end{pmatrix}
					$$
				</section>

				<section data-markdown>
					<script type="text/template">
						### Mathjax support with markdown

						$$
						\begin{pmatrix}
						1 & 2 \\\
						3 & 4
						\end{pmatrix}
						$$

					</script>
				</section>

This is probably a problem of markdown parsing.

Many thanks for such a nice work!

**Comment by hakimel** [comment]
This was fixed by @gothma in #1205. The change is only in the dev branch so far but will go out with the next reveal.js version.

For it to work you'll need to wrap TeX in a code block using backticks ``` `$\sum_{m} \sum_{d}$` ```
